test/c++/runtime_lib_test.cpp

```diff
@@ -776,7 +776,7 @@ TEST_F(RuntimeLibTest, IntersectSortedNodeSetOneSetEmpty){
     auto A = new NodeID[0]{};
     auto B = new NodeID[5]{3, 4, 23, 45, 56};
 
-    ASSERT_DEATH(intersectSortedNodeSetHiroshi(A, B, 0, 5), ".*");
+    //ASSERT_DEATH(intersectSortedNodeSetHiroshi(A, B, 0, 5), ".*");
```

Comment:
What are these asserts for?

Reply:
It was previously testing if Hiroshi errors out when the two list sizes are less than 3. But i changed it so that it just calls the naive intersection. So we don't need this assert anymore